user/libpinyin/template.py

```diff
@@ -0,0 +1,35 @@
+pkgname = "libpinyin"
+pkgver = "2.9.92"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = [
+    "--with-dbm=KyotoCabinet", 
+    "--enable-libzhuyin"
+]
+hostmakedepends = [
+    "pkgconf", 
+    "autoconf", 
+    "automake", 
+    "libtool",
+    "wget2",
+    "gettext"
+]
+
+makedepends = [
+    "kyotocabinet-devel", 
+    "glib-devel"
+]
+    
+depends = ["pcre2-devel"]
+pkgdesc = "Algorithms core for intelligent sentence-based Chinese pinyin IMEs"
+maintainer = "metalparade <comer@live.cn>"
+license = "GPL-3.0-or-later"
+url = "https://github.com/libpinyin/libpinyin"
+#do not use this url, packages fetched have no model data files, which need to be fetched during build stage, however we don't have network access
+#source = f"{url}/archive/refs/tags/{pkgver}.tar.gz" 
```

Comment:
```suggestion
```

this is already apparent by the fact the build would fail, so it doesn't need the comment